summarization/extractor_output_preprocessor.py

```diff
@@ -79,7 +86,9 @@ def _split_content(self):
         '''
         for content in self.content_list:
             if content.content_type == ContentType.TEXT:
-                if content.type == "title":
+                if (content.type in self.TITLE_CONTENT_TYPES
```

Comment:
Is this not handled in extractor already? Also why?

Reply:
resolving this comment as per discussion.